nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -100,7 +100,9 @@ services.xserver.displayManager.defaultSession = "xfce+icewm";
    <listitem>
     <para>
     UPower's configuration is now managed by NixOS and can be customized
-    via <option>services.upower</option>.
+    via <option>services.upower</option>. Stand-alone usage of
+    <literal>UPower</literal> requires the enabled module instead
+    of just <xref linkend="opt-environment.systemPackages"/>.
```

Comment:
CC @NixOS/nixos-release-managers let me know if you'd prefer this second sentence go in "Backwards Incompatibilies" rather than inlined with the new service description, and feel free to suggest rewording of course.

Reply:
I'd put it in both given that it fixes a segfault; better safe than sorry for people to notice it.

Reply:
Updated accordingly.